includes/wikia/DefaultSettings.php

```diff
@@ -1676,6 +1676,11 @@
  */
 $wgDisableWAMOnHubs = false;
 
+/* @name wgTreatAsCorporate
+ * Treat a wiki as if it was a corporate wiki
+ */
+$wgTreatAsCorporate = false;
```

Comment:
Lets step back for a moment. Our main problem is: which wikis should be displayed in language chooser in footer.
We already found out that putting there corporate pages is not enough, because we have pt-br hub like corporate page. That is why we added pseudohub there.
Now we see that not all pseudohubs should be displayed there. HudlerHub is the reason.

Maybe we should create completely separate logic to put wiki in language chooser? Separate variable and logic completely separated from existing variables. What do you think about it?